### PR TITLE
Version 1.0.0-beta.6

### PR DESCRIPTION
**Pull request by MichaelBengtsson**
* Feature - Add support for WC Gift Cards
* Feature - Add support for PW WooCommerce Gift Cards
* Feature - Add support for YITH WooCommerce Gift Cards
* Feature - Add support for multi-currency and multi-language.
* Feature - Add support for shipping selection in the Dintero Embedded Express checkout.
* Feature - Add a rounding fee to orders where the order amount and order items combined amount differs. This is caused by some incorrect calculations and rounding by WooCommerce.
* Fix - Fixed an issue with using a older version of the branding from Dintero.
* Fix - Fixed a compatibility issue with Porterbuddy.
* Fix - Fixed an issue with sending the incorrect shipping id when capturing an order.
* Fix - Fixed issues regarding the callback handling from Dintero.
* Tweak - Set the redirect form factor as the default.

### CODE REVIEW

File: classes/class-dintero-checkout-callback.php [hunk 1]
```diff
@@ -27,149 +27,180 @@ public function __construct() {
...
+				Dintero_Checkout_Logger::log( 'CALLBACK: Handling AUTHORIZED order status. Maybe triggering payment_complete.' );
+				dintero_confirm_order( $order );
+				break;
+			case 'AUTHORIZATION_VOIDED':
+				Dintero_Checkout_Logger::log( 'CALLBACK: Handling AUTHORIZATION_VOIDED order status. Setting order status to CANCELLED.' );
+				$order->update_status( 'cancelled', __( 'The order was CANCELED in the Dintero.', 'dintero-checkout-for-woocommerce' ) );
+				$order->save();
+				break;
+			case 'CAPTURED':
+				Dintero_Checkout_Logger::log( 'CALLBACK: Handling CAPTURED order status.' );
```
Comment (andersem) [inline classes/class-dintero-checkout-callback.php:125]: Seems some order notes are removed? Weren't they useful?

File: classes/class-dintero-checkout-gateway.php [hunk 2]
```diff
@@ -94,7 +88,7 @@ public function get_icon() {
 		 * @return boolean
 		 */
 		public function is_available() {
-			return ! ( 'yes' !== $this->enabled );
+			return ( 'yes' === $this->enabled );
```
Comment (andersem) [inline classes/class-dintero-checkout-gateway.php:91]: :)

**Review by andersem — APPROVED** [review]

**Comment by andersem** [comment]
@MichaelBengtsson Looks good to me, just curious why some order notes were removed.

**Comment by MichaelBengtsson** [comment]
> @MichaelBengtsson Looks good to me, just curious why some order notes were removed.

Because i had some issues that i wanted to solve with them being printed several times. For example, when i placed an order with Swish i got three callbacks that triggered the notes to be printed each time. So i want to build some logic to make sure that we only handle that once, but did not have time to solve it before today. And since there was issues that Jakob reported a merchant was waiting on a solution for, i did not want to hold up the release for something like that.

**Pull request merged**